config.py

```diff
@@ -986,6 +986,15 @@ def make_pwd_account_address(comps):
         # 'centerline',
         # 'curb',
     ],
+    'geocode_types_on_curb': [
```

Comment:
geocode_types_on_curb currently are not being used . geocode_types_in_street is used in address_summary to prioritize geocode types. I included geocode_types_on_curb for consistency (also tom wanted to add geocode_curb_x/y to address summary so they would be used in that implementation).  If we move away from having multiple geocode types in address summary and just pull them on the fly then we could presumably allow a on_curb flag that will access this dictionary for prioritization.